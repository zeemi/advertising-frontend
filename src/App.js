import React from 'react';
import './App.css';
import Header from './components/Header/Header';
import MetricsDisplay from "./components/MetricsDisplay/MetricsDisplay";

function App() {
  return (
    <div className="App">
      <h1>Adverity Advertising Data ETL-V Challenge</h1>
      <Header/>
      <MetricsDisplay/>
    </div>
  );
}

export default App;
