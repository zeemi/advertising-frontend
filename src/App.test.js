import React from 'react';
import {shallow} from 'enzyme';
import App from './App';

describe('<App>', function () {
  it('should render header and metric display', function () {
    expect(shallow(<App/>)).toMatchSnapshot()
  });
});