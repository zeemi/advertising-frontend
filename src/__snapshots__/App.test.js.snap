// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App> should render header and metric display 1`] = `
<div
  className="App"
>
  <h1>
    Adverity Advertising Data ETL-V Challenge
  </h1>
  <Header />
</div>
`;
